React/CxxBridge/RCTCxxBridge.mm

```diff
@@ -796,7 +796,7 @@ - (void)registerExtraLazyModules
 #endif
 }
 
-- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<id<RCTBridgeModule>> *)modules
+- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<Class> *)modules
```

Comment:
What is class?

Reply:
<img width="1136" alt="Screen Shot 2021-05-17 at 11 36 13 AM" src="https://user-images.githubusercontent.com/48299693/118539281-19785200-b704-11eb-93d6-a31d01c13374.png">

Reply:
Derp, could have done that sorry. So it's intentional that this is a more general array then stuff that conforms to `RCTBridgeModule`, thanks!